databricks/koalas/series.py

```diff
@@ -4324,6 +4324,57 @@ def pct_change(self, periods=1):
 
         return self._with_new_scol((scol - prev_row) / prev_row)
 
+    def combine_first(self, other):
+        """
+        Combine Series values, choosing the calling Series's values first.
+
+        Parameters
+        ----------
+        other : Series
+            The value(s) to be combined with the `Series`.
+
+        Returns
+        -------
+        Series
+            The result of combining the Series with the other object.
+
+        See Also
+        --------
+        Series.combine : Perform elementwise operation on two Series
+            using a given function.
+
+        Notes
+        -----
+        Result index will be the union of the two indexes.
+
+        Examples
+        --------
+        >>> s1 = ks.Series([1, np.nan])
+        >>> s2 = ks.Series([3, 4])
+        >>> s1.combine_first(s2)
+        0    1.0
+        1    4.0
+        Name: 0, dtype: float64
```

Comment:
Seems like this case works differently from pandas'

```python
import numpy as np
import databricks.koalas as ks
import pandas as pd

index = pd.MultiIndex.from_tuples([("a", "b"), ("a", "b"), ("a", "c")])
# s1 = ks.Series([1,2,3], index=index)
# s2 = ks.Series([3,4,6], index=index)
s1 = pd.Series([1,2,3], index=index)
s2 = pd.Series([3,4,6], index=index)
s1.combine_first(s2)
```


```
a  c    3
   b    1
   b    1
   b    2
   b    2
```

We might need to select distict columns.

Reply:
Ah, let me check. thanks for the example